src/engraving/types/typesconv.cpp

```diff
@@ -1143,6 +1143,35 @@ String TConv::toXml(const PitchValue& v)
     return String(u"point time=\"%1\" pitch=\"%2\" vibrato=\"%3\"").arg(v.time).arg(v.pitch).arg(v.vibrato);
 }
 
+struct AccidentalUnicodeItem {
+    AccidentalVal accidental;
+    const char* name = nullptr;
+    const char* fullName = nullptr;
+};
+
+static const std::vector<AccidentalUnicodeItem> ACCIDENTALS_NAMES = {
+    { AccidentalVal::SHARP3, QT_TRANSLATE_NOOP("engraving", "triple ♯"), QT_TRANSLATE_NOOP("engraving", "triple sharp") },
+    { AccidentalVal::SHARP2, QT_TRANSLATE_NOOP("engraving", "double ♯"), QT_TRANSLATE_NOOP("engraving", "double sharp") },
+    { AccidentalVal::SHARP, QT_TRANSLATE_NOOP("engraving", "♯"), QT_TRANSLATE_NOOP("engraving", "sharp") },
+    { AccidentalVal::NATURAL, QT_TRANSLATE_NOOP("engraving", "natural") },
+    { AccidentalVal::FLAT, QT_TRANSLATE_NOOP("engraving", "♭"), QT_TRANSLATE_NOOP("engraving", "flat") },
+    { AccidentalVal::FLAT2, QT_TRANSLATE_NOOP("engraving", "double ♭"), QT_TRANSLATE_NOOP("engraving", "double flat") },
+    { AccidentalVal::FLAT3, QT_TRANSLATE_NOOP("engraving", "triple ♭"), QT_TRANSLATE_NOOP("engraving", "triple flat") }
+};
+
+String TConv::translatedName(AccidentalVal accidental, bool full)
```

Comment:
According to the code, this method corresponds to `userName`. The `translatedName` is when it has already been translated (the result of the mtrc method)

Reply:
Then the string will be pass through the mtrc twice, this can have strange consequences...